Header header
uint32 counter
uint32 sample_time